src/docs/environment/index.mdx

```diff
@@ -17,36 +21,82 @@ You're going to be working out of this repository for the remainder of the setup
+```shell
+brew uninstall docker && brew cask install \
+  https://raw.githubusercontent.com/Homebrew/homebrew-cask/3ae8ea3cabebeb1fc0caababdfc739a910d8e0cf/Casks/docker.rb
+```
+
+One thing that requires manual attention is `docker`, which should have just been installed. Open up Spotlight,
+search for "Docker" and start it. You should soon see the Docker icon in your macOS menubar. Docker will automatically
+run on system restarts, so this should be the only time you do this.
 
-You can verify that Docker is running by running `docker ps` in your terminal. If it doesn't error with something like `Error response from daemon: dial unix docker.raw.sock: connect: connection refused`, you're good to continue.
+You can verify that Docker is running by running `docker ps` in your terminal.
 
 ## Build Toolchain
 
-Sentry depends on [Python Wheels](https://pythonwheels.com/), packages containing binary extension modules. We distribute these wheels for the following platforms:
+Sentry depends on [Python Wheels](https://pythonwheels.com/), packages containing binary extension modules.
+We distribute these wheels for the following platforms:
 
 - Linux compatible with [PEP-513](https://www.python.org/dev/peps/pep-0513/) (`manylinux1`)
 - macOS 10.15.0 or newer
 
-If your development machine does not run one of the above systems, you need to install the Rust toolchain. Follow the instructions at [https://www.rust-lang.org/tools/install](https://www.rust-lang.org/tools/install) to install the compiler and related tools. Once installed, the Sentry setup will automatically use Rust to build all binary modules without additional configuration.
+If your development machine does not run one of the above systems, you need to install the Rust
+toolchain. Follow the instructions at [https://www.rust-lang.org/tools/install](https://www.rust-lang.org/tools/install)
+to install the compiler and related tools. Once installed, the Sentry setup will automatically use Rust to build all
+binary modules without additional configuration.
 
-We generally track the latest stable Rust version, which updates every six weeks. Therefore, ensure to keep your Rust toolchain up to date by occasionally running:
+We generally track the latest stable Rust version, which updates every six weeks. Therefore, ensure to keep your Rust
+toolchain up to date by occasionally running:
 
 ```bash
 rustup update stable
 ```
 
 ## Python
 
-We utilize [pyenv](https://github.com/pyenv/pyenv) to install and manage python versions. It should have already been installed earlier when you ran `brew bundle`.
+**NOTE**: We've recently switched our development environment to use Python 3 by default. Temporarily you will still be able
```

Comment:
Don't we have an actual `Note` component or something we can use here @dcramer ?

Or do we hate those callout boxes, I feel like I've heard that somewhere

Reply:
Yes, grep for stuff like this: `<Alert title="Note" level="info">`.

@armenzg could you switch to using those?